tracing/opentracing/interceptors_test.go

```diff
@@ -82,6 +82,7 @@ func TestTaggingSuite(t *testing.T) {
 	mockTracer := mocktracer.New()
 	opts := []grpc_opentracing.Option{
 		grpc_opentracing.WithTracer(mockTracer),
+		grpc_opentracing.WithWhitelistedErrorCodes(codes.NotFound),
```

Comment:
I find the name a little bit confusing, it's not easily intuitive that this makes it so that `codes.NotFound` doesn't set the error status on the span